hub/scripts.js

```diff
@@ -519,6 +536,7 @@
 
   async function decoratePage() {
     decorateTables();
+    handleImageMetadata();
```

Comment:
maybe rename this to `handleDefaultImageMetadata`

Reply:
updated